src/AltinnCore/Common/Helpers/WorkflowHelper.cs

```diff
@@ -70,7 +70,7 @@ public static string GetUrlForCurrentState(Guid instanceId, string applicationOw
                 case WorkflowStep.Archived:
                     return $"/runtime/{applicationOwner}/{applicationId}/{instanceId}/#Preview";
                 default:
-                    return $"/runtime/{applicationOwner}/{applicationId}/ManualTesting";
+                    return $"/designer/{applicationOwner}/{applicationId}/ManualTesting";
```

Comment:
As this is common code for both studio and app, this must be rewritten later when we know where the app must be redirected